lmdeploy/lite/apis/auto_awq.py

```diff
@@ -28,9 +34,30 @@
     'BaiChuanForCausalLM': 'RMSNorm',  # Baichuan 7B
     'BaichuanForCausalLM': 'RMSNorm',  # Baichuan2 7B
     'LlamaForCausalLM': 'LlamaRMSNorm',
+    'LlavaLlamaForCausalLM': 'LlamaRMSNorm',
+    'MGMLlamaForCausalLM': 'LlamaRMSNorm',  # mini gemini
+    'InternLMXComposer2ForCausalLM': 'InternLM2RMSNorm',
 }
 
 
+def save_vl_model(vl_model, model_path, dst_path):
+    safe_serialization = type(vl_model).__name__ == 'MGMLlamaForCausalLM'
+    vl_model.save_pretrained(dst_path,
+                             max_shard_size='2GB',
+                             safe_serialization=safe_serialization)
+    candidate = [
+        'preprocessor_config.json', 'processor_config.json', 'vit',
+        'generation_config.json'
+    ]
+    for name in candidate:  # TODO not only local but also remote path
+        tmp_path = osp.join(model_path, name)
+        if osp.exists(tmp_path):
```

Comment:
what it would be if `tmp_path` not exists?

Reply:
there will be no copy behavior